services/sky/document_view.cc

```diff
@@ -171,11 +201,15 @@ mojo::ScopedMessagePipeHandle DocumentView::TakeRootBundleHandle() {
 }
 
 mojo::ScopedMessagePipeHandle DocumentView::TakeServicesProvidedToEmbedder() {
-  return services_provided_to_embedder_.PassMessagePipe();
+  // TODO(jeffbrown): Stubbed out until we migrate from native viewport
+  // to a new view system that supports embedding again.
+  return mojo::ScopedMessagePipeHandle().Pass();
```

Comment:
I looked - it doesn't seem to support nullptr init :/.  You can drop the .Pass()